TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementCheckers.cs

```diff
@@ -0,0 +1,23 @@
+﻿using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.Parser;
+using TypeCobol.Compiler.Parser.Generated;
+
+namespace TypeCobol.Compiler.Sql.CodeElements
+{
+    class SqlCodeElementCheckers
+    {
+        public static void OnSavepointStatementChecker(CodeElement codeElement, CodeElementsParser.SavepointStatementContext context)
```

Comment:
To match what already exist in Cobol, please rename to 'OnCodeElement'. First param should be of type `SavepointStatement` though.